classes/01class/exercises/c01-git01/afaryy/README.md

```diff
@@ -0,0 +1,57 @@
+# c01-git01
+
+## Commands executed
+
+1. Commands typed to solve the "Merging" exercise
+```
+--Make a new branch called bugFix and Checkout the bugFix branch
+git checkout -b bugFix
+
+--Commit once
+git commit
+
+--Go back to master with git checkout
+git checkout master
+
+--Commit another time
+git commit
+
+--Merge the branch bugFix into master with git merge
+git merge bugFix
+
+--Merge the branch master into bugFix with git merge
+git checkout bugFix; git merge master
+
+```
+
+2. Commands typed to solve the "Rebasing" exercise
+```
+--Checkout a new branch named bugFix
+git checkout -b bugFix
+
+--Commit once
+git commit
+
+--Go back to master and commit again
+git checkout master; git commit
+
+--Check out bugFix again and rebase onto master
+git checkout bugFix; git rebase master
+
+--Check out master again and rebase onto bugFix
+git checkout master; git rebase bugFix
+
+```
+
+3. Commands typed to solve the "Revert and reset" exercise
+```
+--To reverse the most recent commit on both local(local branch) and pushed(remote branch). 
+git reset HEAD~1
```

Comment:
That's right! We are one commit in front of master so it will go back one. We can also do `git reset master` if you wanted.